nodes/parachain/Cargo.toml

```diff
@@ -19,6 +19,7 @@ substrate-build-script-utils = {git = "https://github.com/paritytech/substrate",
 peregrine-runtime = {path = "../../runtimes/peregrine"}
 runtime-common = {path = "../../runtimes/common"}
 spiritnet-runtime = {path = "../../runtimes/spiritnet"}
+empty-runtime = {path = "../../runtimes/empty"}
```

Comment:
Shouldn't we stick to `void` here to keep it consistent? Else name the chain spec `empty`.

Reply:
yeah noticed halfway through that void is cooler